x/commerciomint/module.go

```diff
@@ -18,13 +18,15 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+const DefaultCreditsDenom = "uccc"
```

Comment:
Why use a constant here? credits_denom is already in genesis state

Reply:
we need it for DefaultGenesis